pkg/scheduler/factory.go

```diff
@@ -221,6 +217,12 @@ func (c *Configurator) createFromPolicy(policy schedulerapi.Policy) (*Scheduler,
 		return nil, err
 	}
 
+	// If profiles is already set, it means the user is using both CC and policy config, error out
+	// since these configs are no longer merged and they should not be used simultaneously.
+	if c.profiles != nil {
```

Comment:
why not do this in options.go like before?

Reply:
This shouldn't really happen.

> why not do this in options.go like before?

Now profiles are getting defaulted in CC defaulting logic, so the CC that we create in options.go will always have profiles set. That is why I am resetting it to nil there if you noticed.